config/routes.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  get "health_check" => "application#health_check"
+  scope module: "public" do
```

Comment:
Enforcing clear namespacing which doesn't add cruft to the user-facing URLs :+1: